docs/primitives/a-sky.md

```diff
@@ -24,14 +30,30 @@ A plain color as a background:
 
 ## Attributes
 
-Note that the sky primitive inherits common [mesh attributes](./mesh-attributes.md).
-
 | Attribute       | Component Mapping       | Default Value |
-|-----------------|-------------------------|---------------|
+| --------        | -----------------       | ------------- |
+| color           | material.color          | #FFF          |
+| metalness       | material.metalness      | 0             |
+| opacity         | material.opacity        | 1             |
+| phi-length      | geometry.phiLength      | 360           |
+| phi-start       | geometry.phiStart       | 0             |
 | radius          | geometry.radius         | 100           |
```

Comment:
The default radius value for a-sky is 5000, right? It was changed back recently: https://github.com/aframevr/aframe/pull/1585